jwst/cube_build/cube_build_step.py

```diff
@@ -27,6 +27,7 @@ class CubeBuildStep (Step):
          band = option('SHORT','MEDIUM','LONG','ALL','short','medium','long','all',default='ALL')
          grating   = option('PRISIM','G140M','G140H','G235M','G235H',G395M','G395H','ALL','all',default='ALL')
          filter   = option('CLEAR','F100LP','F070LP','F170LP','F290LP','ALL','all',default='ALL')
+         pipeline = integer(2,3,default=3)
```

Comment:
I'm not sure I really see the advantage in having the `pipeline` argument instead of just using the `output_type` setting when calling cube_build from the calspec2 pipeline. Either way calspec2 has to use and set a particular cube_build argument (either pipeline or output_type), so why is it preferable to set `pipeline`? IMHO, setting `output_type=multi` is a more direct and obvious way of indicating the desired behavior, rather than setting `pipeline=2`. It's not obvious without reading the code or the documentation exactly what's going to happen when you set `pipeline=2`, whereas `output_type=multi` tells you right away what behavior you're going to get.